composer/theme/package.json

```diff
@@ -2,7 +2,12 @@
   "name": "@ballerina/theme",
   "version": "0.982.1-SNAPSHOT",
   "description": "Theme and font for web components of Ballerina Tooling.",
-  "keywords": ["ballerina", "ballerina-lang", "composer", "theme"],
+  "keywords": [
```

Comment:
Seems like these changes are not related to this PR.

Reply:
Reverted